xwiki-platform-distribution/xwiki-platform-distribution-debian/xwiki-platform-distribution-debian-common/pom.xml

```diff
@@ -51,7 +51,36 @@
     </dependency>
   </dependencies>
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources/deb</directory>
+        <includes>
+          <include>copyright</include>
+        </includes>
+        <filtering>true</filtering>
+        <targetPath>${project.build.directory}/deb</targetPath>
+      </resource>
+    </resources>
     <plugins>
+      <plugin>
+        <!-- Get the current year -->
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>1.12</version>
```

Comment:
This plugins version is already defined in xwiki-commons root pom (like most plugins we use).